src/lib/updating-element.ts

```diff
@@ -11,6 +11,19 @@
  * subject to an additional IP rights grant found at
  * http://polymer.github.io/PATENTS.txt
  */
+import {supportsAdoptedStyleSheets, CSSStyleSheetOrCssText} from './css-tag.js';
+export * from './css-tag.js';
+
+// Augment existing types with bleeding edge API.
+declare global {
```

Comment:
I think we may want these here: https://github.com/Polymer/lit-element/blob/master/src/env.d.ts

Reply:
Added types to `env.d.ts`